### PR TITLE
Mock delineate watershed

### PR DESCRIPTION
**Pull request by caseypt**
* When the user selects one of the delineate watershed methods,
generate a random polygon centered where the user clicked. The
shape occasionally looks vaguely like a watershed. This is
temporary until there is support for generating watershed boundaries.

**To test:**
- Select one of the delineate watershed methods from the dropdown.
- Click a point.
- A random polygon should be added to the map.

Connects to #300.

### CODE REVIEW

**Comment by caseypt** [comment]
I followed the @kdeloach Method for Updating NPM Shrinkwrap :copyright:. Whenever I update it, it seems to switch the `from` attribute from a version reference to a tar file on npmjs.org. Not sure why.

File: src/mmw/js/src/draw/views.js [hunk 1]
```diff
@@ -153,16 +155,28 @@ var PlaceMarkerView = Marionette.ItemView.extend({
...
-                addLayer(shape);
-                navigateToAnalyze();
-            }).fail(function() {
-                revertLayer();
-            });
+            // TODO: This is temporary until we have
+            // endpoints that can actually delienate
+            // watersheds.
+            var point = {
+                  "type": "Feature", "properties": {}, "geometry": {
```
Comment (lliss) [inline src/mmw/js/src/draw/views.js]: Ah, yet another reason I dislike the single var statement. It messes up the nesting for objects like this.

File: src/mmw/package.json [hunk 2]
```diff
@@ -40,6 +40,8 @@
     "sinon": "1.14.1",
     "testem": "0.5.15",
     "turf-area": "1.1.1",
+    "turf-buffer": "^1.0.4",
+    "turf-random": "^1.0.2",
```
Comment (lliss) [inline src/mmw/package.json]: Do we want to lock the version numbers? Seems like we've done that with all the other packages.

**Comment by lliss** [comment]
This is a fun feature. Looks good +1. Tested it on all three possible ways to delineate an area.

**Comment by lliss** [comment]
We should add a note or a task issue to remove turf random from the project when this becomes real.

**Comment by lliss** [comment]
Not sure what's up with the shrikwrap file. I'm at the point of feeling we should stop fighting with it.

File: src/mmw/package.json (hunk 2 above)
Comment (caseypt) [inline src/mmw/package.json]: Good call. Forgot about that.

**Comment by caseypt** [comment]
Created an issue to remove turf-random: https://github.com/WikiWatershed/model-my-watershed/issues/307

**Comment by caseypt** [comment]
Updated `package.json` with pinned versions for the turf libraries.

**Comment by caseypt** [comment]
@azaveaci rebuild.

**Pull request merged**